build.xml

```diff
@@ -71,7 +76,7 @@
           resultproperty="test.result"
           failonerror="true">
       <arg value="-c"/>
-      <arg value=". bin/activate;cd ../extension;cfx test ${cfx.binary}"/>
+      <arg value=". bin/activate;cd ../extension;cfx --verbose test ${cfx.binary}"/>
```

Comment:
From the issue you wrote:
> I would always enable verbose logging for cfx test, but we could make it opt-in for cfx run and similar commands.

Did I misunderstand that?
This enables verbosity not for Travis CI but for local testing too.